README.md

```diff
@@ -125,10 +126,4 @@ In the description of shortcuts below, if two keys are connected with a `+`, the
 
 
 ## Alternatives
-* A [docker image](https://hub.docker.com/r/varun19299/cvi-iitm/) for this set-up (last updated Jan 30th, 2017)
 * A Ubuntu customization dedicated to [robotics](https://github.com/ahundt/robotics_setup)
-
-
-## To Dos 
-- [x] CI
-- [ ] configuring default wifi settings
```

Comment:
Removed since I don't think we're actually gonna do this or that it could/should be done also